rpcs3/rpcs3qt/game_compatibility.h

```diff
@@ -32,11 +32,11 @@ class game_compatibility : public QObject
 private:
 	const std::map<QString, compat_status> Status_Data =
 	{
-		{ "Playable", { 0, "", "#1ebc61", QObject::tr("Playable"),         QObject::tr("Games that can be properly played from start to finish") } },
-		{ "Ingame",   { 1, "", "#f9b32f", QObject::tr("Ingame"),           QObject::tr("Games that either can't be finished, have serious glitches or have insufficient performance") } },
-		{ "Intro",    { 2, "", "#e08a1e", QObject::tr("Intro"),            QObject::tr("Games that display image but don't make it past the menus") } },
-		{ "Loadable", { 3, "", "#e74c3c", QObject::tr("Loadable"),         QObject::tr("Games that display a black screen with a framerate on the window's title") } },
-		{ "Nothing",  { 4, "", "#455556", QObject::tr("Nothing"),          QObject::tr("Games that don't initialize properly, not loading at all and/or crashing the emulator") } },
+		{ "Playable", { 0, "", "#00b45a", QObject::tr("Playable"),         QObject::tr("Games that can be properly played from start to finish") } },
+		{ "Ingame",   { 1, "", "#efaa0e", QObject::tr("Ingame"),           QObject::tr("Games that either can't be finished, have serious glitches or have insufficient performance") } },
+		{ "Intro",    { 2, "", "#d78300", QObject::tr("Intro"),            QObject::tr("Games that display image but don't make it past the menus") } },
+		{ "Loadable", { 3, "", "#df4434", QObject::tr("Loadable"),         QObject::tr("Games that display a black screen with a framerate on the window's title") } },
+		{ "Nothing",  { 4, "", "#415254", QObject::tr("Nothing"),          QObject::tr("Games that don't initialize properly, not loading at all and/or crashing the emulator") } },
```

Comment:
Where did you find these colors?
The current ones are correct:
https://github.com/AniLeo/rpcs3-compatibility/blob/master/config.php#L48

Reply:
You are right; is there some kind of overlay on them on the page? Picking them out gives other results. They don't look the same with the same values (at least from here). But we can revert these.

Reply:
There's no overlay on my side of things. I'm unaware of one on parent website and I can't see any transparent layers on top of the page on Inspect Element as well.